iocore/net/NetEvent.h

```diff
@@ -55,6 +55,9 @@ class NetEvent
   // Close when EventIO close;
   virtual int close() = 0;
 
+  bool has_error() const;
+  void set_error();
```

Comment:
Bikeshed - should be named `set_error_from_socket` - otherwise one wonders where the argument to set the error is.